qiskit_experiments/library/characterization/fine_phase.py

```diff
@@ -0,0 +1,129 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine phase characterization experiment."""
+
+from typing import Tuple, List, Optional
+
+import numpy as np
+
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit.library import CXGate
+from qiskit.providers.backend import Backend
+from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.curve_analysis import ParameterRepr
+from qiskit_experiments.curve_analysis.standard_analysis import ErrorAmplificationAnalysis
+
+
+class FineZI(BaseExperiment):
+    """Fine ZI phase characteriaztion.
```

Comment:
I know we did not do this for the FineZXAmplitude but I wonder if we should start having these experiments have a cross-resonance based name? Or perhaps put them in cross-resonance sub-folder. Thoughts?